test/integration/pod_move_with_quota.go

```diff
@@ -0,0 +1,183 @@
+package integration
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/turbonomic/kubeturbo/test/integration/framework"
+	"github.com/turbonomic/turbo-go-sdk/pkg/proto"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/dynamic"
+	kubeclientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	restclient "k8s.io/client-go/rest"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/turbonomic/kubeturbo/pkg/action"
+	"github.com/turbonomic/kubeturbo/pkg/cluster"
+)
+
+var _ = Describe("Action Execution when quota exists ", func() {
```

Comment:
Does the integration test validate that the temporary quota annotation is removed and the quota value is reverted  after the pod move is finished?

Reply:
It did not, but have added the same now. Have put the changes in a saperate commit for clear reference.